topics/persistence.md

```diff
@@ -212,13 +222,19 @@ and a parent process.
 at the same time it writes the new changes in the old append-only file,
 so if the rewriting fails, we are safe).
 
+  Since Redis 7.0.0, the mechanism here has been changed. We have dropped in-memory buffer and open a new INCR AOF file to continue writing. If the rewriting fails finally, the old BASE and INCR files (if we have) plus this newly opened INCR file represents a full amount of data, so we are safe.
```

Comment:
i think we should split the "how it works" section into two.
one (the first one) mentions redis 7, and the other mentions older versions.
they work completely different, so there's no need to add small notes about the differences.
also, it'll be easier for the reader to just read about the version he's using.

p.s. i see you indented this line with two spaces (i assume you wanted it to be part of the last bullet). but in markdown, the empty line between them will prevent that.

p.p.s, look at the README in this folder for styling guide lines on where we want line breaks to be. (i know it's confusing since the old "code" uses a different guideline)

and let's avoid the upper case terms BASE and INCR. i think they're ok in the code, but not the docs (unless we first establish these terms in some TOC, but i don't think that's needed)

Reply:
OK